packages/core/useMounted/index.ts

```diff
@@ -13,7 +13,7 @@ export function useMounted() {
   if (instance) {
     onMounted(() => {
       isMounted.value = true
-    }, isVue2 ? null : instance)
+    }, isVue2 ? void 0 : instance)
```

Comment:
Isn't it better to use `undefined` and let bundlers do their job by using `void 0` if they want?

It's just my opinion, I prefer to read `undefined`, it's seems clearer to me.